### PR TITLE
Fix/clean up repo

### PR DESCRIPTION
**Pull request by wojtekzyla**

### CODE REVIEW

**Review by omrozowicz-splunk — APPROVED** [review]

File: .github/workflows/ci-build.yaml [hunk 1]
```diff
@@ -50,7 +53,58 @@ jobs:
...
+          images: ghcr.io/splunk/sc4snmp-ui/frontend/container
+          tags: |
+            type=semver,pattern=v{{major}}.{{minor}}
+            type=semver,pattern=v{{major}}
+            type=semver,pattern=v{{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+            type=semver,pattern={{version}}
+            type=ref,event=branch
+            type=ref,event=pr
```
Comment (omrozowicz-splunk) [inline .github/workflows/ci-build.yaml]: not sure if we need to build pr images, I think this is too much

File: .github/workflows/ci-release.yaml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          images: ghcr.io/splunk/sc4snmp-ui/frontend/container
+          tags: |
+            type=semver,pattern=v{{major}}.{{minor}}
+            type=semver,pattern=v{{major}}
+            type=semver,pattern=v{{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+            type=semver,pattern={{version}}
+            type=ref,event=branch
+            type=ref,event=pr
```
Comment (omrozowicz-splunk) [inline .github/workflows/ci-release.yaml]: same thing with pr tag

File: .github/workflows/ci-release.yaml [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+          images: ghcr.io/splunk/sc4snmp-ui/backend/container
+          tags: |
+            type=semver,pattern=v{{major}}.{{minor}}
+            type=semver,pattern=v{{major}}
+            type=semver,pattern=v{{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
+            type=semver,pattern={{version}}
+            type=ref,event=branch
+            type=ref,event=pr
```
Comment (omrozowicz-splunk) [inline .github/workflows/ci-release.yaml]: same here

**Review by omrozowicz-splunk — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-build.yaml (hunk 1 above)
Comment (wojtekzyla) [reply .github/workflows/ci-build.yaml]: changed

File: .github/workflows/ci-release.yaml (hunk 2 above)
Comment (wojtekzyla) [reply .github/workflows/ci-release.yaml]: changed

File: .github/workflows/ci-release.yaml (hunk 3 above)
Comment (wojtekzyla) [reply .github/workflows/ci-release.yaml]: changed

**Review by omrozowicz-splunk — APPROVED** [review]

**Pull request merged**